server/db.js

```diff
@@ -0,0 +1,9 @@
+require('dotenv').config();
+const { createClient } = require('@supabase/supabase-js');
+
+const supabase = createClient(
+  process.env.Supabase_URL,
+  process.env.Supabase_API_Key
```

Comment:
did you update the example.env as well? Last night, it said to use Superbase_URL and Superbase_API_Key

Reply:
yea the example.env should be the same and matches the process.env on the db.js